### PR TITLE
Resolve #20: Update team profile & User Guide

### LINKED ISSUE #2: Update AboutUs page
AboutUs page: This page is used for module admin purposes. Please follow the format closely or else our scripts will not be able to give credit for your work.
Add your own details. Include a suitable photo as described here.
Including the the supervisor/lecturer is optional.
The filename of the profile photo should be doc/images/githbub_username_in_lower_case.png e.g. docs/images/damithc.png. If your photo is in jpg format, name the file as .png anyway.
Indicate the different roles played and responsibilities held by each team member. You can reassign these roles and responsibilities (as explained in Admin  Project Scope) later in the project, if necessary.

### LINKED ISSUE #20: Update User Guide
Update docs/UserGuide.md

### PR DESCRIPTION
**Pull request by plosslaw**
Resolves #20

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,7 +3,8 @@ layout: page
 title: User Guide
 ---
 
-AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). If you can type fast, AB3 can get your contact management tasks done faster than traditional GUI apps.
+McScheduler is a **a one-stop solution for McDonald's Shift Managers to manage shift scheduling and worker contact/compensation, optimized for use via a Command Line Interface** (CLI) while still having the benefits of a Graphical User Interface (GUI). With the McScheduler, all information needed for shift-work assignment is made available easily to streamline the work of McDonald's Shift Managers.
```
Comment (wakululuu) [inline docs/UserGuide.md]: > McScheduler is a **a one-stop solution**

Typo here

File: docs/UserGuide.md [hunk 2]
```diff
@@ -14,23 +15,25 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
...
+   * **`worker add`**`n/John hp/98765432 a/21 Lower Kent Ridge Rd, Singapore 119077 r/Cashier p/7` : Adds a cashier named John whose pay is $7/hr. His phone number is 98765432 and he lives at 21 Lower Kent Ridge Rd, Singapore 119077.
 
-   * **`add`**`n/John Doe p/98765432 e/johnd@example.com a/John street, block 123, #01-01` : Adds a contact named `John Doe` to the Address Book.
+   * **`worker delete`**`3` : Deletes the 3rd worker shown in the worker list.
 
-   * **`delete`**`3` : Deletes the 3rd contact shown in the current list.
+   * **`shift add`**`d/Wed t/AM r/Cashier-2 r/Cleaner-3` : Adds a shift on Wednesday morning, which requires 2 workers to fill the cashier role and 3 workers to fill the cleaner role.
 
-   * **`clear`** : Deletes all contacts.
+   * **`assign`**`s/3 w/2 r/Cashier`: Assign the 2nd worker on the list to the 3rd shift on the list as a Cashier.
```
Comment (wakululuu) [inline docs/UserGuide.md]:
```suggestion
   * **`assign`**`s/3 w/2 r/Cashier` : Assign the 2nd worker on the list to the 3rd shift on the list as a Cashier.
```
Inconsistent spacing around `:` compared to the rest of the bullet points

**Review by wakululuu — CHANGES_REQUESTED** [review]

File: docs/UserGuide.md (hunk 1 above)
Comment (plosslaw) [reply docs/UserGuide.md]: Thanks

File: docs/UserGuide.md (hunk 2 above)
Comment (plosslaw) [reply docs/UserGuide.md]: Thanks

**Review by wakululuu — APPROVED** [review]

**Pull request merged**